Form-Controls/styles.css

```diff
@@ -0,0 +1,41 @@
+:root{
+    --blue:#54ADB6;
+    --black:#000000;
+    --brown:#641E16;
+    --yellow:#CC99FF;
+
+}
+
+body{color: var(--brown);
```

Comment:
Please format your code automatically with Prettier.

When you start collaborating with other developers, it's important that everyone use the same format rules so that "junk changes" don't get into your codebase and make it impossible to see what code/features have really been worked on.